common/lib/xmodule/xmodule/js/src/video/09_bumper.js

```diff
@@ -39,8 +39,10 @@ define('video/09_bumper.js',[], function () {
         },
 
         showMainVideoHandler: function () {
-            this.saveState();
-            this.showMainVideo();
+            setTimeout(function () {
+                this.saveState();
+                this.showMainVideo();
+            }.bind(this), 20);
```

Comment:
@auraz we have to add a small delay to be sure other events are invoked before destroying the bumper.